src/pytypes/onefuzztypes/models.py

```diff
@@ -856,4 +856,9 @@ class Task(BaseModel):
     user_info: Optional[UserInfo]
 
 
+class InstanceConfig(BaseModel):
+    admins: Optional[List[UUID]] = None
```

Comment:
Could we add comments documenting the meaning of these fields? The most important is what it means to be an admin, but also whether pool `modification` encompasses creation and deletion.